java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -455,6 +455,32 @@ public ColumnVector copyToDevice() {
   // DATA ACCESS
   /////////////////////////////////////////////////////////////////////////////
 
+  /**
+   * WARNING: Strictly for test only. This call is not efficient for production.
+   */
+  ColumnBuilder.StructData getStruct(long rowIndex, ColumnBuilder.DataType mainType) {
+    assert rowIndex < rows;
+    assert type == DType.STRUCT;
+    List<Object> retList = new ArrayList<>();
+    int start = offHeap.offsets.getInt(rowIndex * DType.INT32.getSizeInBytes());
```

Comment:
Is a struct and a list some how combined in cudf??? @nvdbaranec 

https://arrow.apache.org/docs/format/Columnar.html#struct-layout shows no offsets in a struct at all.